x/distribution/keeper/hooks.go

```diff
@@ -107,8 +105,5 @@ func (h Hooks) BeforeValidatorModified(_ sdk.Context, _ sdk.ValAddress)
 func (h Hooks) AfterValidatorBonded(_ sdk.Context, _ sdk.ConsAddress, _ sdk.ValAddress)         {}
 func (h Hooks) AfterValidatorBeginUnbonding(_ sdk.Context, _ sdk.ConsAddress, _ sdk.ValAddress) {}
 func (h Hooks) BeforeDelegationRemoved(_ sdk.Context, _ sdk.AccAddress, _ sdk.ValAddress)       {}
-func (h Hooks) UnbondingDelegationEntryCreated(_ sdk.Context, _ sdk.AccAddress, _ sdk.ValAddress, _ int64, _ time.Time, _ sdk.Int, _ uint64) {
-}
-func (h Hooks) BeforeUnbondingDelegationEntryComplete(_ sdk.Context, _ uint64) bool {
-	return false
+func (h Hooks) AfterUnbondingOpInitiated(_ sdk.Context, _ uint64) {
```

Comment:
Why are these changes in this PR? Didn't Jehan already merged this?

Reply:
Correct he did using this branch [finish-staking-hooks-merge](https://github.com/cosmos/cosmos-sdk/tree/finish-staking-hooks-merge) I chose to continue on top of [interchain-security-rebase (https://github.com/cosmos/cosmos-sdk/tree/interchain-security-rebase) bc its name is more self-explanatory

Reply:
@sainoe Should we merge that first to keep the PR's (and thus the commits) contained? It will help in the end when the cosmos-sdk team will need to review everything and merge it into the master branch.